src/routers/login/controller.spec.ts

```diff
@@ -2,37 +2,38 @@ import { expect } from 'chai';
 import * as sinon from 'sinon';
 import { LoginController } from './controller';
 import { Model } from 'mongoose';
-import { UserModel } from "../../models/User";
+import { UserModel } from '../../models/User';
 import { Request, Response } from 'express';
 import * as models from '../../models';
 
+// tslint:disable:no-unused-expression
```

Comment:
When we use chai like:
```
expect(true).to.be.true;
expect(spy.calledWith(300)).to.be.true;
...
or whatever expect() function
```

Throw as error this rule: [no-unused-expression](https://palantir.github.io/tslint/rules/no-unused-expression/)

I tried to use [tslint-no-unused-expression-chai](this https://www.npmjs.com/package/tslint-no-unused-expression-chai) but does not work. Maybe a bad configuration from my side.

What do you think about temporary disable this rule in spec files, and then try to do something more robust?